src/test/getarg_tests.cpp

```diff
@@ -13,10 +13,19 @@
 #include <boost/algorithm/string.hpp>
 #include <boost/test/unit_test.hpp>
 
-BOOST_FIXTURE_TEST_SUITE(getarg_tests, BasicTestingSetup)
+namespace getarg_tests{
+    class LocalTestingSetup : BasicTestingSetup {
+        protected: 
+        void SetupArgs(ArgsManager& m_args, const std::vector<std::pair<std::string, unsigned int>>& args); 
+        void ResetArgs(const std::string& strArg); 
```

Comment:
Might wanna remove these trailing whitespaces to appease Travis

Reply:
Thanks Carl 🙏